src/org/zaproxy/zap/extension/proxies/OptionsProxiesPanel.java

```diff
@@ -58,9 +63,26 @@ public OptionsProxiesPanel(ExtensionProxies extension) {
     private void initialize() {
         this.setName(Constant.messages.getString("proxies.options.title"));
         this.setLayout(new GridBagLayout());
-        this.add(getMainProxyPanel(), LayoutHelper.getGBC(0, 0, 1, 0.0));
-        this.add(getSecurityProtocolsPanel(), LayoutHelper.getGBC(0, 1, 1, 0.0));
-        this.add(getProxiesOptionsPanel(), LayoutHelper.getGBC(0, 2, 1, 1.0, 1.0));
+        this.add(getScrollPane(), LayoutHelper.getGBC(0, 0, 1, 1.0, 1.0));
+    }
+    
+    private JScrollPane getScrollPane() {
+        if (proxiesScrollPane == null) {
+            proxiesScrollPane = new JScrollPane();

```

Comment:
This could use an empty border, `proxiesScrollPane.setBorder(BorderFactory.createEmptyBorder());`.